### PR TITLE
Add bilibili Embed Service

### PR DESCRIPTION
**Pull request by soulteary**
Added the most popular video UGC platform [Bilibili](https://www.bilibili.com/blackboard/aboutUs.html) for young people in China.

image

### CODE REVIEW

**Comment by soulteary** [comment]
Due to the https://github.com/outline/outline/pull/1660 change of the main code, a file merge conflict occurred, and the code was rebase and [force pushed](https://github.com/outline/outline/compare/55136f4adf373fc290aeb93a72190f1b2b999908..54f78b1a043d75cc6319fc889529bc2e3997e562).

**Comment by tommoor** [comment]
Hi @soulteary – the logo image must be 120x120px and to the edges, please see the other images for good examples. I would probably just use the logo as the name will be displayed next to it automatically

File: shared/embeds/index.js [hunk 1]
```diff
@@ -66,6 +67,13 @@ export default [
     component: Airtable,
     matcher: matcher(Airtable),
   },
+  {
+    title: "Bilibili",
+    keywords: "video",
```
Comment (tommoor) [inline shared/embeds/index.js:72]: Include the Chinese in keywords?

**Comment by soulteary** [comment]
> Hi @soulteary – the logo image must be 120x120px and to the edges, please see the other images for good examples. I would probably just use the logo as the name will be displayed next to it automatically

I will update it later today, thanks for reminding

File: shared/embeds/index.js (hunk 1 above)
Comment (soulteary) [reply shared/embeds/index.js:72]: Is it like this? Does it feel strange? I'm not sure what to do.

>     keywords: "video, 哔哩哔哩, 视频弹幕网站",


Are there any references and standards for this type of integration? Maybe there should be a catalog or website about outline development and design specifications.

**Comment by soulteary** [comment]
Icon has been updated.

https://github.com/outline/outline/pull/2550/files?short_path=265bc6c#diff-265bc6cf816cab3b47b5aa171f6d92691abffc213e3233d963e75955bbdcda2a

**Pull request merged**